internal/agent/device/config/config_test.go

```diff
@@ -0,0 +1,61 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestReplaceTokensInArgs(t *testing.T) {
+	require := require.New(t)
+	tests := []struct {
+		name     string
+		args     []string
+		tokens   map[string]string
+		expected []string
+	}{
+		{
```

Comment:
Can you add tests that tokenMap contains key other than "FilePath" ?

Reply:
I can be they are not supported directly. This does need more thought

Reply:
passes now